airflow/www/templates/airflow/dags.html

```diff
@@ -170,12 +173,16 @@ <h2>{{ page_title }}</h2>
                   <a></a>
                   <span aria-hidden="true" title=" " class="material-icons text-muted js-tooltip" style="display:none">info</span>
                 </td>
-                {# Column 7: Recent Tasks #}
+                {# Column 7: Next Run #}
+                <td class="text-nowrap next-run">
```

Comment:
We've been using a `js-` prefix for classes used for scripting (not styling) purposes.
```suggestion
                <td class="text-nowrap js-next-run">
```